.devcontainer/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM mcr.microsoft.com/vscode/devcontainers/javascript-node:16
+
+ENV PS_INSTALL_FOLDER=/opt/microsoft/powershell
+ENV PSModuleAnalysisCachePath=/var/cache/microsoft/powershell/PSModuleAnalysisCache/ModuleAnalysisCache
+
+RUN mkdir -p ${PS_INSTALL_FOLDER} \
+    && (curl -L https://github.com/PowerShell/PowerShell/releases/download/v7.1.3/powershell-7.1.3-linux-x64.tar.gz --output - \
+        | tar xzf - -C ${PS_INSTALL_FOLDER}) \
+    && chmod a+x,o-w ${PS_INSTALL_FOLDER}/pwsh \
+    && ln -s ${PS_INSTALL_FOLDER}/pwsh /usr/bin/pwsh \
+    && ln -s ${PS_INSTALL_FOLDER}/pwsh /usr/bin/pwsh-lts \
+    && export POWERSHELL_TELEMETRY_OPTOUT=1 \
+    && pwsh \
+        -NoLogo \
+        -NoProfile \
+        -Command " \
```

Comment:
I believe you should be able to use `'` strings here to avoid escaping.

Reply:
Although I don't understand why this is necessary.

Reply:
This is copied directly from the microsoft base dockerfiles for powershell